src/util.ts

```diff
@@ -651,7 +652,18 @@ export class Util {
           activeRequest_ = util.makeRequest(
             authenticatedReqOpts!,
             reqConfig,
-            callback!
+            // tslint:disable-next-line:only-arrow-functions
```

Comment:
You can use an arrow function and rest params to do this!
```js
(apiResposeError, ...params) => {
  // ....
  callback!(apiResponseError, ...params);
```

Reply:
Forehead slap.